examples/iris.py

```diff
@@ -0,0 +1,26 @@
+import gym
+from urdfenvs import iris
+import numpy as np
+
+
+def main():
+    env = gym.make("iris-rotor-vel-v0", dt=0.01, render=True)
+    n_steps = 1000
+    pos0 = np.array([-1.0, 0.2, -0.0])
+    ob = env.reset(pos=pos0)
```

Comment:
This is currently not affecting the initial position. Probably related to `n=3`, see below.

Reply:
Setting both the initial position and pose is a good suggestion. I will fix this today.